canonical/ubuntu.com
# Add a redirect for the realtime engage page

## Done

- Add a redirect for the realtime engage page

## QA

- Open the demo
- Go to /engage/an-introduction-to-real-time-linux-part-i/
- Check it redirects to /engage/an-introduction-to-real-time-linux
